packages/api/db/migration/20220715214935_sensor_as_standard_location.js

```diff
@@ -0,0 +1,56 @@
+const deleteSensorData = require('../deleteSensorData');
+
```

Comment:
@smattingly this file as well.

Reply:
This does not autopopulate the uuid for the sensor table-- not sure if that's intentional. Otherwise, this look fine to me.

Reply:
@smattingly I imagine that the primary key for the sensor table (`location_id`) will be generated by the location table. I believe the other locations (ex. field, gate) use the same logic for their primary keys.

Also, should we keep the farm_id in the sensor table?

Reply:
I didn't think of that, and it makes sense, although I checked some other location "subtype" tables that do autogenerate.

Good question about farm_id. For consistency with the rest of the design, I would lean toward "no".